XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +341,10 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func changeBrowserView(view: String) {
```

Comment:
I think having this as an `enum` in `FxScreenGraph` would be more suitable here.

```swift
enum BrowserMenuChoice: String {
   case Settings = "…"

}
```